### PR TITLE
Add query argument to Workflow.get_runs()

### PR DESCRIPTION
**Pull request by thburghout**
Adds missing argument `created`, `exclude_pull_requests`, `head_sha` to the query options of `Workflow.get_runs()`.

First time contributing. Please let me know if something needs to be done differently. Some manual editing in the replay data was needed as `check.yml` does not exists any more.

Solves #2337

### CODE REVIEW

**Comment by thburghout** [comment]
Hi @JLLeitschuh, what would be the best way to move this forward? Can I update the pull request to get the review going?

**Comment by JLLeitschuh** [comment]
> Can I update the pull request to get the review going?

That would be appropriate. The PR looks good to me, sans conflicts that need to be resolved.

File: github/Workflow.py [hunk 1]
```diff
@@ -137,13 +137,15 @@ def get_runs(
         branch=github.GithubObject.NotSet,
         event=github.GithubObject.NotSet,
         status=github.GithubObject.NotSet,
+        head_sha=github.GithubObject.NotSet,
     ):
         """
         :calls: `GET /repos/{owner}/{repo}/actions/workflows/{workflow_id}/runs <https://docs.github.com/en/rest/reference/actions#workflow-runs>`_
```
Comment (EnricoMi) [inline github/Workflow.py:143]: Please update the docs link to https://docs.github.com/en/rest/actions/workflow-runs#list-workflow-runs-for-a-workflow

**Review by EnricoMi — CHANGES_REQUESTED** [review]
While you are extending this method, can you also add the following missing parameters?
- created
- exclude_pull_requests
- check_suite_id

in the order listed at https://docs.github.com/en/rest/actions/workflow-runs#list-workflow-runs-for-a-workflow.

**Comment by EnricoMi** [comment]
Please run `tox` locally and commit the changes.

**Comment by EnricoMi** [comment]
Looks like `check_suite_id` has been added in master in the meanwhile.

**Comment by thburghout** [comment]
@EnricoMi @JLLeitschuh this PR is be ready to be reviewed. If there is anything still needed, please let me know.

**Review by EnricoMi — APPROVED** [review]
LGTM!

**Pull request merged**